data/siteMetadata.js

```diff
@@ -1,6 +1,7 @@
 module.exports.siteMetadata = {
   title: 'Tailscale Community',
-  description: 'Tailscale how-tos, solutions, and integrations',
+  description:
+    "The official community site for developers building with Tailscale. Find how-tos, solutions, and integrations to make your network's security the easy part.",
```

Comment:
> ...to build your own secure network you can reach from anywhere.

yay? nay?

Reply:
> ...to build your own secure network that just works.

A bit tongue in cheek, but the company's motto is "the secure network that just works".